LondonBritishLibrary/orient/BLorient788.xml

```diff
@@ -37,64 +39,159 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                                        <height>251</height>
-                                        <width>235</width>
+                                    <measure unit="leaf">4+165+1</measure>
+                                    <note>Four unnumbered blank leaves at the beginning and one at the end.</note>
+                                    <dimensions type="outer" unit="in">
+                                        <height>9.88</height>
+                                        <width>9.25</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
                             </supportDesc>
                             
                             <layoutDesc>
-                                <layout columns="2" writtenLines="20 21" cert="high"/>
+                                <layout columns="2" writtenLines="20 21"/>
                             </layoutDesc>
                         </objectDesc>
                         
                         <handDesc>
-                            
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is well written of the <date>18th century</date>. 
-                                    
-                                </desc>
-                                
-                                <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
-                                
+                                <date notBefore="1700" notAfter="1799" resp="PRS10171Wright"/>
+                                <desc>Well written.</desc>
                             </handNote>
-                            
                         </handDesc>   
+                        
                         <additions>
-                            
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#165r"/>
+                                    <desc type="OwnershipNote"></desc>
+                                    <q xml:lang="gez">ዝመጽሐፍ፡ ዘቅዱስ፡ ሩፋኤል፡ ፈውስ፡ መንፈሳዊ፤ ዘምስለ፡ ቀንዲል፤ ወሥርዓተ፡ ቤተ፡ ክርስቲያን። <foreign xml:lang="am">የፍትሐት፡ መጽሐፍ፤ ምስለ፡ ተክሊል፤ የሌሊትና፡
+                                        የመዓልት፡ ሰዓታት፤ ምስለ፡ ጸሎተ፡ ዕጣን፤ ያዓመት፡ እግዜር፡ ነግሥ፤ ተአምረ፡ እግዝእትነ፡ ማርያም፤</foreign> እሉ፡ ኵሎሙ፡ ዘወሀቦሙ፡ <persName ref=""><roleName>መምህሬ፡</roleName> ኢሳይያስ፡</persName> ገብሩ፡ ለሩፋኤል። 
+                                        ዘሰረቆ፡ ወዘሤጣመ፤ ወዘተሣየጦሙ፤ በሥልጣነ፡ ጴጥሮስ፡ ወጳውሎስ፡ ውጉዘ፡ ለይኩን። ወበ፫፻፲ወ፰ጳጳሳት፡ ውጉዘ፡ ይኩን፡ ከመ፡ አርዮስ።</q>
+                                </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#159r"/>
+                                    <desc type="Record">Amharic deed of the time of <persName ref="PRS3692egwalas">Gʷālu</persName>.</desc>
```

Comment:
`<date notBefore="1801" notAfter="1818">of the time of <persName ref="PRS3692egwalas">Gʷālu</persName></date>`

Reply:
I don't see if the date tag is closed
should not be either `<date notBefore="1801" notAfter="1818"/>` or `</date>` somewhere?

Reply:
;)
(I think that in cases of such obvious typos, it is faster for all if the person who spots them corrects the file immediately)